### PR TITLE
Perfect the non Chinese part

### PR DESCRIPTION
**Pull request by Kojima648**

### CODE REVIEW

File: mods/braindance_protocol/lang/cn.lua [hunk 1]
```diff
@@ -1,646 +1,843 @@
...
+    button_language                                             = "简中" ,
     parent_Cheats                                               = "作弊" ,
     parent_Crafting                                             = "制作图纸" ,
-    parent_Cyberware                                            = "Cyberware Items" ,
-    parent_Facts                                                = "Facts [Spoiler Alert!]" ,
+    parent_Cyberware                                            = "赛博义体" ,
+    parent_Facts                                                = "游戏事实 [剧透警告!]" ,
     parent_ItemMods                                             = "道具修改" ,
     parent_ItemSets                                             = "道具生成" ,
+    -- parent_PlayerStats                                          = "玩家数据" ,
```
Comment (Nats-ji) [inline mods/braindance_protocol/lang/cn.lua]: 这里还有下面那些Line的注释需要取消掉。

File: mods/braindance_protocol/lang/cn.lua [hunk 2]
```diff
@@ -1,646 +1,843 @@
...
-    item_Facts_GoroFate_btn1                                    = "Dead" ,
-    item_Facts_GoroFate_btn2                                    = "Alive" ,
-    item_Facts_SetFingersToFriendly_name                        = "Set Fingers To Friendly" ,
-    item_Facts_SetFingersToFriendly_tip                         = "Fingers M.D. is now friendly." ,
+    item_Facts_SkippyMode_name                                  = "设置四季宝模式" ,
+    item_Facts_SkippyMode_tip                                   = "设置四季宝为 \"冷血杀手\" 模式或 \"爱狗人士\" 模式." ,
+    item_Facts_SkippyMode_btn1                                  = "杀手" ,
+    item_Facts_SkippyMode_btn2                                  = "爱狗" ,
+    item_Facts_SkippyAttitude_name                              = "设置四季宝态度" ,
+    item_Facts_SkippyAttitude_tip                               = "设置四季宝为 \"友善\" or \"生气\"." ,
```
Comment (Nats-ji) [inline mods/braindance_protocol/lang/cn.lua]: 这里的or可以改成中文吗

File: mods/braindance_protocol/lang/cn.lua [hunk 3]
```diff
@@ -1,646 +1,843 @@
...
-    parent_Teleport2                                            = "传送 - Generic" ,
+    parent_Teleport                                             = "传送 - V的公寓" ,
+    parent_Teleport2                                            = "传送 - 常用" ,
     parent_Teleport3                                            = "传送 - NPCs" ,
-    parent_Teleport4                                            = "传送 - Stores" ,
-    parent_Teleport5                                            = "传送 - Ripperdocs" ,
-    parent_Teleport6                                            = "传送 - Bonus Locations" ,
+    parent_Teleport4                                            = "传送 - 商店" ,
+    parent_Teleport5                                            = "传送 - 义体医生" ,
+    parent_Teleport6                                            = "传送 - 额外附加地点" ,
```
Comment (Nats-ji) [inline mods/braindance_protocol/lang/cn.lua:18]: 这里应该改成塔罗牌地点

**Review by Nats-ji — CHANGES_REQUESTED** [review]
Thank you for the translation. Good work! I've made some change suggestions. Can you take a look at them?

**Review by Kojima648 — COMMENTED** [review]
已修改

**Comment by Nats-ji** [comment]
谢谢！

**Pull request merged**